edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesEnvironmentOperator.cs

```diff
@@ -75,24 +75,24 @@ async Task OnListPodsCompleted(Task<HttpOperationResponse<V1PodList>> task)
         void HandlePodChangedAsync(WatchEventType type, V1Pod pod)
         {
             // if the pod doesn't have the module label set then we are not interested in it
-            if (!pod.Metadata.Labels.TryGetValue(Constants.K8sEdgeModuleLabel, out string podName))
+            if (!pod.Metadata.Labels.ContainsKey(Constants.K8sEdgeModuleLabel))
             {
                 return;
             }
 
-            Events.PodStatus(type, podName);
+            Events.PodStatus(type, pod.Metadata.Name);
```

Comment:
lets make it similar to other log methods
```suggestion
            Events.PodStatus(type, pod);
```